src/index.html

```diff
@@ -8,6 +8,58 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <main>
+      <div class="container">
+
+        <div class="stars--0 stars">
```

Comment:
it's ok, but usually modifiers like `stars--0` we use after block `stars`
```suggestion
        <div class="stars stars--0">
```